src/shared/components/common/language-select.tsx

```diff
@@ -51,13 +51,14 @@ export class LanguageSelect extends Component<LanguageSelectProps, any> {
     ) : (
       <div>
         {this.props.multiple && this.props.showLanguageWarning && (
-          <div className="alert alert-warning" role="alert">
+          <div className="language-select alert alert-warning" role="alert">
```

Comment:
Why would this alert `div` be `language-select`? It should probably be `language-select-alert`.

Reply:
Belonged on parent element, fixed.